packages/phone-number-privacy/common/src/utils/authentication.ts

```diff
@@ -0,0 +1,73 @@
+import { ContractKit } from '@celo/contractkit'
+import { AuthenticationMethod } from '@celo/contractkit/lib/identity/odis/query'
+import { AccountsWrapper } from '@celo/contractkit/lib/wrappers/Accounts'
+import { trimLeading0x } from '@celo/utils/lib/address'
+import { retryAsyncWithBackOff } from '@celo/utils/lib/async'
+import { verifySignature } from '@celo/utils/lib/signatureUtils'
+import { ec as EC } from 'elliptic'
+import { Request } from 'firebase-functions'
```

Comment:
If we're only using this for the type, consider hard-coding in a type def and removing this dependency

Reply:
Which would also help because the Signer's request type may be slightly different (uses Express's request instead of firebase-function's, which are almost identical though)

Reply:
I changed it to use express instead so it's more compatible with both the combiner and signer, thanks!